### ISSUE TITLE
error on destroy

### ISSUE THREAD

**Issue by pgte** [architect/architect#1351]
When using `arc-destroy` I'm getting an error `× ValidationException: 1 validation error detected: Value '[...]' at 'names' failed to satisfy constraint: Member must have length less than or equal to 10`, apparently on deleting SSM parameters.
```
$ cd apps/backend && ../../node_modules/.bin/arc-destroy  --app decipad-backend --name 1368 --force --no-timeout
...
s3rver storing data in /tmp/.s3rver_data/NrZlMZFtD4ZXfvAjKKXpt
         App ⌁ decipad-backend
      Region ⌁ eu-west-2
     Profile ⌁ Set via environment
     Version ⌁ Destroy 3.0.3
         cwd ⌁ /home/runner/work/decipad/decipad/apps/backend

⚬ Destroy Destroying staging environment
⚬ Destroy Reminder: if you deployed to production, don't forget to run destroy again with: --production
⚬ Destroy Destroying DecipadBackendStaging1368 in 5 seconds...
⚬ Destroy Destroying DecipadBackendStaging1368
⚬ Destroy Deleting static S3 bucket...
⚬ Destroy Retrieving deployment bucket...
⚬ Destroy Deleting deployment S3 bucket...
⚬ Destroy Deleting SSM parameters...
× ValidationException: 1 validation error detected: Value '[/decipad-backend/staging/DECI_FROM_EMAIL_ADDRESS, /decipad-backend/staging/DECI_KEY_VALIDATION_EXPIRATION_SECONDS, /decipad-backend/staging/DECI_S3_ACCESS_KEY_ID, /decipad-backend/staging/DECI_S3_ENDPOINT, /decipad-backend/staging/DECI_S3_PADS_BUCKET, /decipad-backend/staging/DECI_S3_SECRET_ACCESS_KEY, /decipad-backend/staging/GITHUB_CLIENT_ID, /decipad-backend/staging/GITHUB_CLIENT_SECRET, /decipad-backend/staging/JWT_SIGNING_PRIVATE_KEY, /decipad-backend/staging/NODE_OPTIONS, /decipad-backend/staging/DECI_APP_URL_BASE, /decipad-backend/staging/DECI_INVITE_EXPIRATION_SECONDS, /decipad-backend/staging/DECI_SES_ACCESS_KEY_ID, /decipad-backend/staging/DECI_SES_SECRET_ACCESS_KEY, /decipad-backend/staging/GIT_COMMIT_HASH, /decipad-backend/staging/JWT_SECRET, /decipad-backend/staging/NEXTAUTH_URL, /decipad-backend/deploy/bucket]' at 'names' failed to satisfy constraint: Member must have length less than or equal to 10.
[31](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:32)
ValidationException: 
[32](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:33)
    at Request.extractError (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/protocol/json.js:52:27)
[33](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:34)
    at Request.callListeners (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
[34](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:35)
    at Request.emit (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
[35](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:36)
    at Request.emit (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/request.js:688:14)
[36](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:37)
    at Request.transition (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/request.js:22:10)
[37](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:38)
    at AcceptorStateMachine.runTo (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/state_machine.js:14:12)
[38](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:39)
    at /home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/state_machine.js:26:10
[39](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:40)
    at Request.<anonymous> (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/request.js:38:9)
[40](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:41)
    at Request.<anonymous> (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/request.js:690:12)
[41](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:42)
    at Request.callListeners (/home/runner/work/decipad/decipad/node_modules/@architect/destroy/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
[42](https://github.com/decipad/decipad/runs/6693010808?check_suite_focus=true#step:6:43)
error Command failed with exit code 1.
```

**Comment by ryanblock** [comment]
Ah, hmm! Looks like we need to be paginating SSM deletion. Will see if I can slip this fix into Arc 10.4!

**Comment by ryanblock** [comment]
Ok, fixed in https://github.com/architect/destroy/pull/21/commits/54f7b14e621d6d6c1e71af563e5788ba112c45c0

This will ship in 10.4 this week!
